### PR TITLE
[ISSUE-822] Deprecate inline volumes

### LINKED ISSUE #822: Deprecate support of inline volumes
**Is your feature request related to a problem? Please describe.**
Need to deprecate inline volumes support. Generic ephemeral volumes must be used instead.

**Describe the solution you'd like**
No support for inline volumes will be provided.

**Describe alternatives you've considered**
Generic ephemeral volumes.

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by mishoyama**
### Resolves #822 

Deprecate inline volumes feature. Use generic ephemeral volumes instead.

## PR checklist
- [x] Add link to the issue
- [x] Choose Project
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [x] All comments are resolved

## Testing
CI passed

### CODE REVIEW

File: tests/e2e/scenarios/bare-metal-testdriver.go [hunk 1]
```diff
@@ -84,7 +84,8 @@ func initBaremetalDriver() *baremetalDriver {
 
 var _ storageframework.TestDriver = &baremetalDriver{}
 var _ storageframework.DynamicPVTestDriver = &baremetalDriver{}
-var _ storageframework.EphemeralTestDriver = &baremetalDriver{}
+// todo add test coverage for generic ephemeral volumes
```
Comment (safronovD) [inline tests/e2e/scenarios/bare-metal-testdriver.go:87]: Maybe skip simple [EphemeralVolume ](https://github.com/kubernetes/kubernetes/blob/7f5380a4035edb083cf163647158a05b12057157/test/e2e/storage/framework/testpattern.go#L130) testpattern [here ](https://github.com/dell/csi-baremetal/blob/f5e629aa544f90310f3ca39455be3442e72c77d4/tests/e2e/scenarios/bare-metal-testdriver.go#L125). Generic Ephemeral Volumes should become supported automatically

**Review by safronovD — APPROVED** [review]

File: tests/e2e/scenarios/bare-metal-testdriver.go (hunk 1 above)
Comment (mishoyama) [reply tests/e2e/scenarios/bare-metal-testdriver.go:87]: Thanks, will try

**Review by mitsaa — APPROVED** [review]

File: tests/e2e/scenarios/bare-metal-testdriver.go (hunk 1 above)
Comment (mishoyama) [reply tests/e2e/scenarios/bare-metal-testdriver.go]: Done

**Review by rkiselenko — APPROVED** [review]

**Pull request merged**